commcare-cloud/commcare_cloud/commands/ansible/ansible_playbook.py

```diff
@@ -229,3 +235,84 @@ def run(self, args, unknown_args):
         args.playbook = 'deploy_stack.yml'
         unknown_args += ('--tags=users',)
         AnsiblePlaybook(self.parser).run(args, unknown_args)
+
+
+class ManageServices(AnsibleOptions, _AnsiblePlaybookAlias):
+    """
+    example usages
+    1. To restart riak and stanchion only for riakcs
+       only option can be skipped to restart all services which are a part of riakcs
+       i.e riak, riak-cs and stanchion in that order
+        commcare-cloud staging service riakcs restart --only riak stanchion riak-cs
+    2. To start services under proxy i.e nginx
+       You can still pass ansible options which are
+       --user, --become, --become_user
+        commcare-cloud staging service proxy start -b
```

Comment:
I would push for _not_ allowing `--user`, `--become`, `--become_user`, etc. or at least not requiring them for proper functioning; rather I think this command itself should know which user to run each command under and pass in `--become`, etc., accordingly. (I think in every case it's just `root`, so it would mean always passing in `--become`.)

You can do that with `args.become = True` before the call to `RunAnsibleModule`. In addition if you take this suggetion the way I'm imaging, you won't need the `AnsibleOptions` change.